src/Microsoft.AspNetCore.Blazor.Build/targets/Blazor.MonoRuntime.props

```diff
@@ -25,13 +25,4 @@
     <BlazorBootJsonOutputPath>$(BaseBlazorRuntimeOutputPath)$(BlazorBootJsonName)</BlazorBootJsonOutputPath>
   </PropertyGroup>
 
-  <ItemGroup>
-    <MonoBCLFile Include="$(MonoBaseClassLibraryPath)*.dll" />
-    <MonoBCLFacadeFile Include="$(MonoBaseClassLibraryFacadesPath)*.dll" />
-    <MonoAsmjsFile Include="$(MonoAsmjsRuntimePath)**/*.*" />
-    <MonoWasmFile Include="$(MonoWasmRuntimePath)**/*.*" />
-    <BlazorJsFile Include="$(BlazorJsPath)" />
-    <MonoBaseClassLibraryFolder Include="$(MonoBaseClassLibraryPath);$(MonoBaseClassLibraryFacadesPath)" />
-  </ItemGroup>
```

Comment:
These were being evaluated too early. It only worked before as a random fluke of arbitrary build ordering. It no longer works following other build cleanups elsewhere. So now I've moved them to be evaluated later at a more suitable time.